02_bubble/amitkrishna/bubble.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, bubble([]int{3, 2, 1, 5}))
+}
+
+func bubble(s []int) []int {
+
```

Comment:
Remove unnecessary blank lines at beginning and end of blocks.

Reply:
Removed the blank lines